lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -17,11 +17,31 @@ export default function LMSGrader({
   children,
   assignmentName,
   courseName,
-  students,
+  students: unorderedStudents,
 }) {
   // No initial current student selected
   const [currentStudentIndex, setCurrentStudentIndex] = useState(-1);
 
+  // A sorted list of students. Students are sorted by
+  // displayName.
+  const [students] = useState(() => {
```

Comment:
I think `useMemo` may work here, but `useState`, as we discussed in Slack, is probably not whatcha want.